File: OnJeong_BE/src/main/java/com/a503/onjeong/domain/group/dto/GroupUserListDTO.java
package com.a503.onjeong.domain.group.dto;

import lombok.Builder;
import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class GroupUserListDTO {
    private Long groupId;
    private Long ownerId;
    private String name;
    private List<Long> userList;

    @Builder
    public GroupUserListDTO(
            Long groupId,
            Long ownerId,
            String name,
            List<Long> userList
    ) {
        this.groupId = groupId;
        this.name = name;
        this.ownerId = ownerId;
        this.userList = userList;
    }
}

File: OnJeong_FE/app/src/main/java/com/a503/onjeong/domain/mypage/dto/GroupUserListDTO.kt
package com.a503.onjeong.domain.mypage.dto

import lombok.Builder
import lombok.Data


@Data
class GroupUserListDTO @Builder constructor(
    private val groupId: Long,
    private val ownerId: Long,
    private val name: String,
    userList: List<Long>
) {
    private var userList: List<Long> = ArrayList<Long>()

    init {
        this.userList = userList
    }
}

